gcp_variant_transforms/transforms/variant_to_bigquery.py

```diff
@@ -65,6 +73,7 @@ def __init__(
       variant_merger=None,  # type: variant_merge_strategy.VariantMergeStrategy
       proc_var_factory=None,  # type: processed_variant.ProcessedVariantFactory
       append=False,  # type: bool
+      update_schema=False,  # type: bool
```

Comment:
nit: consider renaming this to `update_schema_on_append` here and everywhere. `update_schema` is generic and b/c this logic is tied to append, I prefer to have the flag and variable names be specific about it.